csrc/executor.cpp

```diff
@@ -236,15 +237,20 @@ void FusionExecutor::debugCompileFusionFromStr(
   }
 
   compiled_kernel_ =
-      executor_utils::getCompiledKernel(std::nullopt, code, name, fusion_id_);
-  NVF_ERROR(fusion_id_ > 0, "assign a fusion_id_ <= 0 is not accepted.");
+      executor_utils::getCompiledKernel(std::nullopt, code, name, kernel_id_);
+  NVF_ERROR(fusion_id_ >= 0, "assign a fusion_id_ < 0 is not accepted.");
```

Comment:
why are we asserting `fusion_id_` here? shouldn't this one be `kernel_id_` instead, since the `getCompiledKernel` is updating that one?